commons/src/main/com/mbrlabs/mundus/commons/assets/AssetType.java

```diff
@@ -31,6 +31,7 @@ public enum AssetType {
     MODEL("Model"),
     /** Terra file. Contains height data for terrains. */
     TERRAIN("Terrain"),
+    TERRAIN_LAYER("TL"),
```

Comment:
Why did you add this line into AssetType? As I see this enum type is not used.

Reply:
Mistake will be removed on next commit.